packages/edge-login-ui-rn/src/native/components/screens/existingAccout/ConfirmPasswordRecoveryScreen.js

```diff
@@ -0,0 +1,65 @@
+// @flow
```

Comment:
Generally speaking, there should be a newline after each `// @flow` statement. Otherwise, the `import-sort` utility can mess this up.